articles/api/authentication/_logout.md

```diff
@@ -24,50 +32,62 @@ curl --request GET \
+| `id_token_hint` (Recommended)         | Previously issued ID Token for the user. This is used to indicate which user to log out                                                                         |
+| `logout_hint` (Optional)              | Optional `sub` (user ID) or `sid` (session ID) value to indicate which user to log out.                                                                         |
+| `post_logout_redirect_uri` (Optional) | URL to redirect the user after the logout.                                                                                                                      |
+| `client_id` (Optional)                | The `client_id` of your application.                                                                                                                            |
+| `federated` (Optional)                | Add this query string parameter to the logout URL, to log the user out of their identity provider, as well: `https://YOUR_DOMAIN/oidc/logout?federated`.        |
+| `state` (Optional)                    | An opaque value the applications adds to the initial request that the authorization server includes when redirecting the back to the`post_logout_redirect_uri`. |
+| `ui_locales` (Optional)               | Space-delimited list of locales used to constrain the language list for the request. The first locale on the list must match the enabled locale in your tenant  |
 
 
 ### Test with Authentication API Debugger
 
 <%= include('../../_includes/_test-this-endpoint') %>
 
-1. At the *Configuration* tab, set the fields **Application** (select the application you want to use for the test) and **Connection** (the name of the social connection to use).
+1. At the Configuration tab, set the fields **Application** (select the application you want to use for the test) and **Connection** (the name of the social connection to use).
 
-1. Copy the <dfn data-key="callback">**Callback URL**</dfn> and set it as part of the **Allowed Logout URLs** of your [Application Settings](${manage_url}/#/applications).
+1. Copy the **_Callback URL_** and set it as part of the **Allowed Logout URLs** of your Application Settings.
 
-1. At the *Other Flows* tab, click **Logout**, or **Logout (Federated)** to log the user out of the identity provider as well.
+1. At the _Other Flows_ tab, click **Logout**, or **Logout (Federated)** to log the user out of the identity provider as well.
 
 
 ### Remarks
 
 - Logging the user out of their identity provider is not common practice, so think about the user experience before you use the `federated` query string parameter.
-- The sample auth0.js script uses the library version 8. If you are using auth0.js version 7, please see this [reference guide](/libraries/auth0js/v7).
+- If providing both `id_token_hint` and `logout_hint`, the `logout_hint` value must match either the `sub` or `sid` claim from the `id_token_hint`.
```

Comment:
- If providing both `id_token_hint` and `logout_hint`, the `logout_hint` value must match either the `sub` or `sid` claim from the `id_token_hint`.
- If providing both `id_token_hint` and `client_id`, the `client_id` value must match either the `aud` claim from the `id_token_hint`.